app/invocation/invocation.tsx

```diff
@@ -280,7 +280,11 @@ export default class InvocationComponent extends React.Component<Props, State> {
     const runnerExecution = this.state.runnerExecution;
     if (!runnerExecution?.executionId) return;
 
-    this.runnerExecutionStream = waitExecution(runnerExecution.executionId, {
+    const service = rpcService.getRegionalServiceOrDefault(
+      this.state.model?.stringCommandLineOption("remote_cache") ?? ""
```

Comment:
We won't have an invocation model at this point unfortunately, because the runner execution itself is responsible for creating the invocation and we stream the execution status just so that we can see the runner execution progress before the invocation gets created. Once we have `this.state.model` we render a nicer page.

In practice we can always use the default service because workflows are only enabled in us-west1 (for now)

Reply:
(I think it's fine to leave this code actually - probably just need to change remote_cache to remote_executor though)

Reply:
i'm kinda caught between doing a nearly-useless change and at least keeping the code flagged as something that needs to care about regions.  i guess i'm gonna just go for it here--that way if we ever do have a bit more data in state we can update here.

My understanding is right that there's nothing dangling on the execution right now that would otherwise identify the region (i.e., we just have the digest and don't know what server that lives on?)

Reply:
yeah we don't have any region identifiers AFAIK - we could in theory add region to the Executions table though.